user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,188 @@
+
+1. Click the *New* button to add another shape data with label `Component` and value `Spectrum`
+
+1. Click the OK button in the `Define Shape Data` window. This will close the window
+
+1. Make sure you have no shapes selected
+
+1. Save your Visio file.
+
+In DataMiner Cube, you will now get a basic spectrum control, with all the features of the spectrum control in the element, but without the ribbon. This control will also display the info pane, which is not desired in the visual overview for our use case.
+
+## Step 5: Create a preset to hide the info pane
+
+1. Open the spectrum simulation element and navigate to the `Spectrum Analyzer` page if needed
+
+1. In the `View` tab of the ribbon, click the `Info pane` button to disable the info pane
+
+1. In the spectrum sidebar, click the `Presets` tab and click the `Manage` button at the bottom.
+
+1. Click the `New...` button
+
+1. Give the preset a name of your choice, e.g. `NoInfoPane`
+
+1. Enable the `Share preset with other users` checkbox
+
+1. Click the `OK` button
+
+1. Click the `OK` button once more to close the Manage presets window
+
+## Step 6: Use the preset on the spectrum component
+
+1. In DataMiner Cube, Navigate back to the `Visio spectrum presets` view and bring Visio to the front from your taskbar. If you have closed Visio previously, you can re-open it by right-clicking in the view card and choosing `Edit in Visio`from the menu.
+
+1. Select the rectangular shape that is linked to the spectrum componen, and in the shape data pane right-click and select `Define Shape Data`
+
+1. Click the "New" button to add an additional shape data field
+
+1. Change the Label to "ComponentOptions"
+
+1. In the value, type `preset=NoInfoPane`
+
+1. Make sure you have no shapes selected
+
+1. Save your Visio file.
+
+In DataMiner Cube, you will now get the same spectrum control, but the info pane is no longer displayed.
+
+## Step 7: Create presets for the measurements
+
+We will now create additional presets for each of the center frequencies where we want to retrieve the frequency spectrum
+
+1. Open the spectrum simulation element and navigate to the `Spectrum Analyzer` page if needed
+
+1. In the `Manual` tab of the spectrum sidebar, change the settings to the desired values. e.g. for a satellite signal this could be around 12GHz center frequency and 200MHz Bandwith
+
+1. In the spectrum sidebar, click the `Presets` tab and click the `Manage` button at the bottom.
+
+1. Click the `New...` button
+
+1. Give the preset a name of your choice, e.g. `Satellite`
+
+1. Enable the `Share preset with other users` checkbox
+
+1. Click the `OK` button
+
+1. Click the `OK` button once more to close the Manage presets window
+
+1. Repeat the above steps for a second signal. E.g. For a cable signal, we could use a center frequency of 260 MHz and a Bandwith of 8 MHz. Give it the name `Cable`
```

Comment:
What I miss here a bit, is maybe the explanation that if in the connector you set the Simulation Mode to "Carriers from Table" mode (see help in here: https://catalog.dataminer.services/details/6f33ec9f-e83d-49d5-8f85-87ad66eaa5c7 ), you can nicely make a realistic simulation value. But it could also be that you explicitly do not want to mention too much that you are working on a Simulation and therefore you did not do it?

Reply:
In this way you could make a little bit the link with KATA #32. But I can also imagine you maybe did not want to give the feeling that the other KATA is a "prerequisite" and therefore it was not done.

Reply:
But I still think it's good to mention it, at least in the tutorial, in case they want to play around it nicely and make it a bit more realistic, that with this mode, they will have a nicer switching animation when they switch between the buttons.

Reply:
I've added a reference to the documentation of the connector. I want to focus on the visio integration here, not the capabilities of the simulation connector. (This could be a topic for a future kata or tutorial)